exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -153,12 +156,14 @@ class _CustomCollector:
     https://github.com/prometheus/client_python#custom-collectors
     """
 
-    def __init__(self):
+    def __init__(self, disable_target_info: bool = False):
```

Comment:
This changes the default behavior of `_CustomCollector`, is it required that exporting resources on target info is enabled by default?

Reply:
To be fair, I don't know. I did not find any specific documentation on this default behavior. 

I therefore decided to use the same behavior as the Prometheus Exporter in Go (enabled by default): https://github.com/open-telemetry/opentelemetry-go/blob/main/exporters/prometheus/exporter.go

Reply:
Would you prefer this metric to be disabled by default?

Reply:
Ah, I see, ok, then.